packages/astro-rss/README.md

```diff
@@ -299,25 +144,69 @@ const item = {
 };
 ```
 
-#### `enclosure.url`
+- `enclosure.url` is the URL where the media can be found. If the media is hosted outside of your own domain you must provide a full URL.
+- `enclosure.length` is the size of the file found at the `url` in bytes.
+- `enclosure.type` is the [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) for the media item found at the `url`.
 
-Type: `string (required)`
+### stylesheet
 
-The URL where the media can be found. If the media is hosted outside of your own domain you must provide a full URL.
+Type: `string (optional)`
 
-#### `enclosure.length`
+An absolute path to an XSL stylesheet in your project. If you don’t have an RSS stylesheet in mind, we recommend the [Pretty Feed v3 default stylesheet](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl), which you can download from GitHub and save into your project's `public/` directory.
 
-Type: `number (required)`
+### customData
 
-The size of the file found at the `url` in bytes.
+Type: `string (optional)`
 
-#### `enclosure.type`
+A string of valid XML to be injected between your feed's `<description>` and `<item>` tags. This is commonly used to set a language for your feed:
 
-Type: `string (required)`
+```js
+import rss from '@astrojs/rss';
+
+export const GET = () => rss({
+    ...
+    customData: '<language>en-us</language>',
+  });
+```
 
-The [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) for the media item found at the `url`.
+### xmlns
 
-## `rssSchema`
+Type: `Record<string, string> (optional)`
+
+An object mapping a set of `xmlns` suffixes to strings of metadata on the opening `<rss>` tag.
```

Comment:
So two things here:

1. A second sentence like "This is used to/for" with plain language explanation of the purpose here would be nice.

2. If the URL is just an example, then standard practice is to use `example.com` EVERYWHERE, even when other URLs seem fun.

Reply:
This took me down a rabbithole to understand what `xmlns` even is. Learned a lot! Updated the entry to give a practical example